### DISCUSSION TITLE
How do I connect to several databases at once?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [pallets-eco/flask-sqlalchemy#1064] [Q&A] [answered]
How do I connect to several databases at once?

**Comment by ThiefMaster** [comment, marked as answer]
look at sqlalchemy "binds". i think the flask-sqlalchemy docs also mention this somewhere

**Comment by martinenkoEduard** [comment]
https://flask-sqlalchemy.palletsprojects.com/en/2.x/binds/
